packages/e/espeak-ng/package.yml

```diff
@@ -1,11 +1,13 @@
 name       : espeak-ng
 version    : '1.51'
-release    : 6
+release    : 7
 source     :
     - https://github.com/espeak-ng/espeak-ng/archive/refs/tags/1.51.tar.gz : f0e028f695a8241c4fa90df7a8c8c5d68dcadbdbc91e758a97e594bbb0a3bdbf
+homepage   : https://github.com/espeak-ng/espeak-ng
 license    :
-    - GPL-3.0
+    - Apache-2.0
     - BSD-2-Clause
+    - GPL-3.0
```

Comment:
Oh sorry i didn't notice this before but `GPL-3.0` is a deprecated SPDX licence identifier. Should be either `GPL-3.0-or-later` or `GPL-3.0-only`.

From https://github.com/espeak-ng/espeak-ng/blob/master/src/espeak-ng.c#L6 it looks like it's `GPL-3.0-or-later`